drivers/npmx/npmx_shell.c

```diff
@@ -570,6 +570,83 @@ static int cmd_charger_trickle_set(const struct shell *shell, size_t argc, char
 	return 0;
 }
 
+static int cmd_charger_discharging_current_get(const struct shell *shell, size_t argc, char **argv)
+{
+	ARG_UNUSED(argc);
+	ARG_UNUSED(argv);
+
+	npmx_instance_t *npmx_instance = npmx_driver_instance_get(pmic_dev);
+
+	if (npmx_instance == NULL) {
+		shell_error(shell, "Error: shell is not initialized.");
+		return 0;
+	}
+
+	npmx_charger_t *charger_instance = npmx_charger_get(npmx_instance, 0);
+	uint16_t current;
+	npmx_error_t err_code = npmx_charger_discharging_current_get(charger_instance, &current);
+
+	if (check_error_code(shell, err_code)) {
+		shell_print(shell, "Value: %d mA.", (int)current);
```

Comment:
Can it be negative? If no, then:
```suggestion
		shell_print(shell, "Value: %u mA.", current);
```